### ISSUE TITLE
Not working from Chrome Store

### ISSUE THREAD

**Issue by PerfectFaro** [shatting/simplenote#15]
Other people have mentioned this, but this isn't available from the Chrome webstore: there is still a problem with an invalid package. Are there plans to fix this, or has the project been abandoned?

**Comment by gsingh93** [comment]
There are forks that have fixed this issue: https://github.com/ocorreiododiogo/simplenote

You'll have to install it from source though, but that isn't very hard. Just google for it.

**Comment by PerfectFaro** [comment]
That was EXACTLY what I needed! Thanks so much!

**Comment by PerfectFaro** [comment]
Actually, I spoke too soon: I couldn't find any info on Google on how to install an extension that isn't .crx. Have you gotten it to work this way, and if so, how did you do it?

**Comment by gsingh93** [comment]
I haven't installed it yet, but I develop chrome extensions so I do know how to install it. Download the files from Github (using Git or as a zip), unzip them if necessary, and then follow these instructions: https://developer.chrome.com/extensions/getstarted#unpacked

**Comment by PerfectFaro** [comment]
Thanks for the instructions, but the extension still has an error, and won't load. The instructions you linked to were correct, but the extension itself is broken. If you do try to install it and get it to work, please let me know.

**Comment by gsingh93** [comment]
You're right, that fork didn't fix all of the issues. I've forked it myself here and fixed all of the issues I saw: https://github.com/gsingh93/simplenote. Try that version.

**Comment by PerfectFaro** [comment]
That worked perfectly! Thanks so much for your help! This is a really great extension, and will be immensely helpful to me. Thanks again!

**Comment by JessicaC** [comment]
I installed that version successfully but if I try to update any of the notes, they aren't saved at all :/

**Comment by JessicaC** [comment]
@gsingh93 
Doesn't work. I mean, it installs fine but the notes aren't saving.

**Comment by JessicaC** [comment]
HELLO??? Is anyone going to help??

**Comment by JessicaC** [comment]
still waiting, months later..
